### PR TITLE
Selenium Grid Helm Chart

### PR DESCRIPTION
**Pull request by ilpianista**
Add an Helm Chart to deploy Selenium Grid.

### Description
Continuation of @pedrodotmc's work in #1184 to support an Helm Chart to deploy Selenium Grid.

### Motivation and Context
Easily deploy Selenium Grid on Kubernetes.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by tgupta-adc** [comment]
When can we expect it to get merged? My team has been waiting for it.

File: chart/selenium-grid/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+This chart enables the creation of a Selenium grid server in Kubernetes.
+
+## Installing the chart
+
+To install the selenium-grid helm chart, you can run:
+
+```bash
+# Clone the project
+git clone https://github.com/pedrodotmc/docker-selenium.git
```
Comment (diemol) [inline chart/selenium-grid/README.md]:
```suggestion
git clone https://github.com/seleniumhq/docker-selenium.git
```

**Review by diemol — COMMENTED** [review]
Thanks for this PR, @ilpianista!

I left a small suggestion.

Do you know how we could test this in GitHub Actions? 

I messaged @pedrodotmc in Slack and Twitter. I think we really need them to sign the CLA to accept the contribution. Will wait a bit until they reply.

**Comment by diemol** [comment]
Also, @ilpianista, would you have time to help us triage issues related to the Helm chart? And eventually join the project?

**Comment by ilpianista** [comment]
> Also, @ilpianista, would you have time to help us triage issues related to the Helm chart? And eventually join the project?

We are using this Helm in my company and thus we are interested in fixing any bug that affects it. I don't know about adding new features though because that depends exclusively on my free time.

**Comment by pedromctech** [comment]
I just signed the CLA. Sorry for the delay, thanks @ilpianista for making this possible!

**Review by diemol — APPROVED** [review]
Thank you, @ilpianista and @pedrodotmc!

**Comment by diemol** [comment]
> > Also, @ilpianista, would you have time to help us triage issues related to the Helm chart? And eventually join the project?
> 
> We are using this Helm in my company and thus we are interested in fixing any bug that affects it. I don't know about adding new features though because that depends exclusively on my free time.

@ilpianista any chance you can join our Slack/IRC/Matrix channel so we can coordinate? Links can be seen here https://www.selenium.dev/support/

**Pull request merged**